test/output/callbacks/callbacks_test.py

```diff
@@ -20,6 +20,7 @@
 
 import sys
 from io import BytesIO, StringIO
+import unittest
```

Comment:
Would prefer a different way to fix this, but we don't enforce lint rules strictly in openhtf so okay for now